packages/yourloops/pages/patient/teams/add-dialog.tsx

```diff
@@ -156,8 +155,8 @@ export function EnterIdentificationCode(props: EnterIdentificationCodeProps): JS
   return (
     <React.Fragment>
       <Box textAlign="center" p={3}>
-        <DialogTitle id="team-add-dialog-title">
-          <strong>{teamName ? t('modal-add-medical-specific-team', { careteam: teamName }) : t('modal-add-medical-team')}</strong>
+        <DialogTitle>
+          <strong id="team-add-dialog-title">{teamName ? t('modal-add-medical-specific-team', { careteam: teamName }) : t('modal-add-medical-team')}</strong>
```

Comment:
I think it was fine before, don't you think?

Reply:
I don't know why but id on mui 5 `DialogTitle` is not shown in the final HTML 🤷🏻‍♂️. If root `Dialog` has an `aria-labelledby` this one is set in title instead, otherwise it generates a random id...